### DISCUSSION TITLE
what does the uBlockOrigin community think about THE "supermium browser" ?

### DISCUSSION THREAD

**Discussion by wireconnectbypassnat** [uBlockOrigin/uBlock-issues#3315] [General]
i am asking because google plans to get rid of mv2 support 

and if i understand correctly, i **soft block** of mv2 will start in the summer of 2024, 

but it can be bypassed by registry policy

and in the **summer of 2025** hard block will be by removing the mv2 code from Chromium

and brave can not be trusted because they sell NFT

supermium browser is the only browser that plans to support mv2 in 2025

**Comment by krystian3w** [comment]
Somewhere in the discussions there is a full description of how to extend the life of the MV2 by a year using policies: https://github.com/uBlockOrigin/uBlock-issues/discussions/2977#discussioncomment-8134456

Try starting with this and freezing use Supremium until June 2025.

**Reply by garry-ut99** [reply]
Read again, as OP is already aware of the policy as he already mentioned about it in his initial comment and he was asking about a browsers stituation **after** the additional policy workaround expires:
> wireconnectbypassnat : but it can be bypassed by registry policy and in the **summer of 2025** hard block will be by removing the mv2 code from Chromium

**Reply by krystian3w** [reply]
He don't liked only Brave, love Google Chrome.

**Reply by garry-ut99** [reply]
Unrelated to what I said, read again.

**Comment by garry-ut99** [comment]
> wireconnectbypassnat : supermium browser is the only browser that plans to support mv2 in 2025

Now Thorium too: 
- https://github.com/uBlockOrigin/uBlock-issues/discussions/3334#discussioncomment-10302576
